typings/index.d.ts

```diff
@@ -38,3 +38,27 @@ declare module 'react-syntax-highlighter/dist/cjs/prism-light';
 declare module 'monaco-editor/esm/vs/basic-languages/markdown/markdown';
 declare module 'monaco-editor/esm/vs/basic-languages/css/css';
 declare module 'monaco-editor/esm/vs/basic-languages/yaml/yaml';
+
+// Storybook uses this module and its types are defined in the source but not in the type output
+declare module 'file-system-cache' {
```

Comment:
Is it possible to keep these definitions isolated in `packages/kbn-storybook/src/typings.ts`?

Reply:
That didn't work, and I don't think I can import here or it will convert from an ambient definition... At this point I'm not sure but those two files are already very similar so maybe it's not a big deal...